app/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,10 @@
     <string name="cancel">Cancel</string>
     <!-- A 'Save' string to be used for labeling UI elements. -->
     <string name="save">Save</string>
+    <!-- An 'Add' string to be used for labeling UI elements. -->
+    <string name="add_label">Add</string>
```

Comment:
Curious if we can just name these "add" and "edit" for the string name?

Reply:
@gabrielluong There already is a string for saved logins named just "edit" and I wanted to have a separate resource from that one.

Reply:
@mcarare oh i see. I am wondering now if we should've just used that one and moved it in with the generic string. I did a quick search for `R.string.edit` and didn't see it being used except in a snackbar test, which makes it generic enough.